terraform/cloud_function/scheduled/main.tf

```
@@ -61,7 +61,7 @@ resource "google_cloud_scheduler_job" "scheduler_job" {
   attempt_deadline = each.value.attempt_deadline != null ? each.value.attempt_deadline : "320s"
   name             = each.value.name
   schedule         = each.value.schedule
-  time_zone        = "Europe/Amsterdam"
+  time_zone        = "Etc/UTC"
```

```suggestion
  time_zone        = "UTC"
```
This is the same thing and easier to read

---

I couldnt find a list of supported values but found this was the default, if UTC works too happy with that